tests/test_adaptor_strict.cc

```diff
@@ -409,6 +411,92 @@ namespace rascal {
     }
   }
 
+  /* ---------------------------------------------------------------------- */
+  using Fixtures_no_center = boost::mpl::list<
+      MultipleStructureFixture<MultipleStructureManagerNLFixtureNoCenter>,
+      MultipleStructureFixture<
+          MultipleStructureManagerNLStrictFixtureNoCenter>>;
+
+  BOOST_FIXTURE_TEST_CASE_TEMPLATE(multiple_no_center_test, Fix,
+                                   Fixtures_no_center, Fix) {
+    bool verbose{false};
+    auto & managers = Fix::managers;
+    int n_it{static_cast<int>(managers.size())};
+    for (int i_it{0}; i_it < n_it; i_it += 2) {
+      auto & manager = managers[i_it];
+      auto & manager_no_center = managers[i_it + 1];
+      auto is_center_atom = extract_underlying_manager<0>(manager_no_center)
```

Comment:
While I understand what is done here, it seems a little convoluted. Is there no data structure in the fixture to get this? Or do you specifically want to test for this exact thing?

Reply:
is_center_atom is accessible only from manager center, so you need to extract it.

Reply:
I see, because afterwards all center atoms are centers and the others are not...